external/docker-integration-tests/pom.xml

```diff
@@ -127,28 +127,11 @@
       <version>11.2.0.1.0</version>
       <scope>test</scope>
     </dependency>
-
-    <!-- DB2 JCC driver manual installation instructions
-
-       You can build this datasource if you:
-        1) have the DB2 artifacts installed in a local repo and supply the URL:
-          -Dmaven.repo.drivers=http://my.local.repo
-
-        2) have a copy of the DB2 JCC driver and run the following commands :
-          mvn install:install-file -Dfile=${path to db2jcc4.jar} \
-            -DgroupId=com.ibm.db2 \
-            -DartifactId=db2jcc4 \
-            -Dversion=10.5 \
-            -Dpackaging=jar
-
-       Note: IBM DB2 JCC driver is available for download at
-          http://www-01.ibm.com/support/docview.wss?uid=swg21363866
-     -->
     <dependency>
       <groupId>com.ibm.db2.jcc</groupId>
-      <artifactId>db2jcc4</artifactId>
-      <version>10.5.0.5</version>
-      <type>jar</type>
+      <artifactId>db2jcc</artifactId>
+      <version>db2jcc4</version>
+      <scope>test</scope>
```

Comment:
IBM Data Server Driver for JDBC and SQLJ is a pure-Java driver (Type 4) that supports the JDBC 4 specification. You can use this JDBC driver for Java applications that access the Db2® LUW database server:
https://mvnrepository.com/artifact/com.ibm.db2.jcc/db2jcc/db2jcc4